src/condor_negotiator.V6/matchmaker.cpp

```diff
@@ -6065,7 +6065,7 @@ void Matchmaker::RegisterAttemptedOfflineMatch( ClassAd *job_ad, ClassAd *startd
 
 
 	classy_counted_ptr<ClassAdMsg> msg = new ClassAdMsg(MERGE_STARTD_AD,update_ad);
-	classy_counted_ptr<DCCollector> collector = new DCCollector();
+	classy_counted_ptr<DCCollector> collector = new DCCollector("");
```

Comment:
it looks like we only use the `const char *` here in one place, a D_HOSTNAME `dprint()`:
```dprintf( D_HOSTNAME, "DCCollector::relocate(%s)\n", this->constructorName.c_str() );```
Should we just ditch this parameter entirely?

Reply:
Yeah, but all that code does is pass the constructor name through to `self` for that error message.

Reply:
The `constructorName.c_str()`  is the 1st argument to 

```
DCCollector::DCCollector( const char* dcName, UpdateType uType )
    : Daemon( DT_COLLECTOR, dcName, NULL )
```

which doesn't do much with `dcName` but I'm worried about leaving foot guns behind.

Reply:
Ah, I'd missed that.  May be clean-up for later.  Thanks.